### ISSUE TITLE
'micro-txn-authorization-response' event not triggered

### ISSUE THREAD

**Issue by Kamadake** [greenheartgames/greenworks#184]
Apologies for leaving another issue here but I wanted to ask. I'm trying to use this event to figure out when I might be able to close the steam overlay after triggering. the problem is this event isn't getting triggered at all. I'm authorizing the transaction through the steam overlay but I'm not seeing this event get triggered. I tried testing this with a simple snippet like:
`greenworks.on('micro-txn-authorization-response', function(app_id, ord_id, authorised) {
      console.log('Hello');
    });`

Is there something extra that maybe I need to do before attach a function to the event?

**Comment by hokein** [comment]
Are you using electron? If so, #61 may solve your problem.

**Comment by Kamadake** [comment]
Yeah I'm using electron, took a look and tried to do the following:

`greenworksInit: function () {
    process.activateUvLoop();
    greenworks.init();

    greenworks.on('micro-txn-authorization-response', function(app_id, ord_id, authorised) {
      console.log('Hello');
    });
  },`

still didn't trigger it though 😢 i'm running this in the main process not at renderer level. could that affect it?

**Comment by hokein** [comment]
@bgSosh do you know any other reason that may cause `micro-txn-authorization-response` not triggered?

**Comment by aftokinito** [comment]
Same problem here with `requestUserInformation` and `persona-state-changed`.
The `process.activateUvLoop()` hackfix doesn't seem to be working anymore.

**Comment by weilitao** [comment]
Same problem here. `process.activateUvLoop()` gotchas not work.

**Comment by Kosmoon** [comment]
For some reason, it only works if you launch the game after uploading your build to steamworks. You need to launch the real build steam provides you
